packages/browser-wallet/src/popup/pages/Account/Account.tsx

```diff
@@ -51,16 +53,29 @@ function Account() {
     const selectedAccount = useAtomValue(selectedAccountAtom);
     const { pathname } = useLocation();
     const nav = useNavigate();
+    const [detailsExpanded, setDetailsExpanded] = useState(true);
 
     const canClose = pathname !== absoluteRoutes.home.account.path;
 
     return (
         <div className="flex-column justify-space-between align-center h-full relative">
+            <MenuButton
+                className="account-page__hide"
+                open={detailsExpanded}
+                onClick={() => setDetailsExpanded((o) => !o)}
+            />
             <div className="account-page__content">
                 {accounts.length === 0 && <div>{t('noAccounts')}</div>}
                 {selectedAccount !== undefined && (
                     <>
-                        <div className="account-page__details">{selectedAccount}</div>
+                        <div
+                            className={clsx(
+                                'account-page__details',
+                                detailsExpanded && 'account-page__details__expanded'
```

Comment:
And correspondingly:

```suggestion
                                detailsExpanded && 'account-page__details--expanded'
```